src/sidebar/components/Annotation/AnnotationHeader.js

```diff
@@ -72,23 +72,15 @@ function AnnotationHeader({
 
   const isCollapsedReply = isReply(annotation) && threadIsCollapsed;
 
-  const annotationIsPrivate = isPrivate(annotation.permissions);
-
   // Link (URL) to single-annotation view for this annotation, if it has
   // been provided by the service. Note: this property is not currently
   // present on third-party annotations.
   const annotationUrl = annotation.links?.html || '';
 
-  const showTimestamps = !isEditing && annotation.created;
   const showEditedTimestamp = useMemo(() => {
     return hasBeenEdited(annotation) && !isCollapsedReply;
   }, [annotation, isCollapsedReply]);
 
-  const replyPluralized = replyCount > 1 ? 'replies' : 'reply';
-  const replyButtonText = `${replyCount} ${replyPluralized}`;
-  const showReplyButton = replyCount > 0 && isCollapsedReply;
-  const showExtendedInfo = !isReply(annotation);
```

Comment:
In some cases I find these abstractions don't end up being helpful and having the conditionals in situ where they're used is easier to human-parse. This represents me changing over time :)